lib/project_types/script/layers/domain/extension_point.rb

```diff
@@ -4,14 +4,14 @@ module Script
   module Layers
     module Domain
       class ExtensionPoint
-        attr_reader :type, :beta, :deprecated, :sdks, :domain
+        attr_reader :type, :beta, :deprecated, :libraries, :domain
 
         def initialize(type, config)
           @type = type
           @beta = config["beta"] || false
           @deprecated = config["deprecated"] || false
           @domain = config["domain"] || nil
-          @sdks = ExtensionPointSDKs.new(config["sdks"])
+          @libraries = ExtensionPointlibraries.new(config["libraries"])
```

Comment:
typo - missing capital letter
```suggestion
          @libraries = ExtensionPointLibraries.new(config["libraries"])
```